qcfractal/interface/client.py

```diff
@@ -622,6 +627,23 @@ def add_collection(
         payload = {"meta": {"overwrite": overwrite}, "data": collection}
         return self._automodel_request("collection", "post", payload, full_return=full_return)
 
+    def delete_collection(self, collection_type: str, name: str) -> None:
```

Comment:
This seems reasonable difficult to do which is a good thing. Do we need to add additional security measures do you think to prevent accidental deletion?